scripts/bundles/compiler.ts

```diff
@@ -81,6 +82,7 @@ export async function compiler(opts: BuildOptions) {
     },
     plugins: [
       typescriptSourcePlugin(opts),
+      relativePathPlugin('@sys-api-node', '../sys/node/index.js'),
```

Comment:
Yeah we were previously using the `aliasPlugin` for that purpose but although the alias plugin is included in several Rollup bundles previously the `@sys-api-node` alias was only actually used inside of the `sys/` bundle. In this diff I make a few changes to also import that module from within `src/compiler` too, so the mapping we were previously doing in the alias plugin won't work anymore. That just mapped `@sys-api-node` to `./index.js', which would resolve correctly in `sys/node` but obvs not in the compiler bundle.

Anyhow - this made me realize I should just map `@sys-api-node` to the full file path in `aliasPlugin` to avoid having to specify relative paths from other bundles, so just made that change.